WirePublisher becomes unusable after configuring wrong CloudPublisher Target Filter

If you put something which is not a valid OSGi target filter in the CloudPublisher Target Filter field the WirePublsher stops working even  after you correct the mistake and put a valid OSGi filter pointing at an existing CloudPublisher.  
Note that if you put a valid filter that is not pointing to anything it doesn't break. The WirePublisher only breaks with an invalid filter.

Steps to reproduce the behavior:
1. Create a CloudPublisher
2. Create a Wire Graph containing a WirePublisher
3. Go to the WirePublisher configurations and put an invalid filter in the CloudPublisher Target Filter field 
4. Apply changes
5. Change WirePublisher configuration to correct one 
6. Apply changes

The WirePublisher should then function properly (send asset data via mqtt) and instead doesn't do anything. If you remove and readd the WirePublisher  without configuring it wrong first then it works.

Kura running on Raspberry Pi 2 
Kura version :  KURA_4.2.0-SNAPSHOT

Actually it does not appear to be the same issue.
Th OSGi console outputs this when changing the configuration to the wrong one: 
```
osgi> 
!ENTRY org.eclipse.kura.wire.component.provider 4 0 2019-12-16 13:27:10.851
!MESSAGE [org.eclipse.kura.wire.CloudPublisher(77)] Invalid syntax in target property for dependency CloudPublisher to pippo

!ENTRY org.eclipse.equinox.cm 4 0 2019-12-16 13:27:10.856
!MESSAGE unexpected InvalidSyntaxException: Missing ')' at "pippo)": (&(objectClass=org.eclipse.kura.cloudconnection.publisher.CloudPublisher)pippo)
!STACK 0
java.lang.RuntimeException: unexpected InvalidSyntaxException: Missing ')' at "pippo)": (&(objectClass=org.eclipse.kura.cloudconnection.publisher.CloudPublisher)pippo)
	at org.apache.felix.scr.impl.manager.ServiceTracker.open(ServiceTracker.java:220)
	at org.apache.felix.scr.impl.manager.ServiceTracker.open(ServiceTracker.java:180)
	at org.apache.felix.scr.impl.manager.DependencyManager.setTargetFilter(DependencyManager.java:2105)
	at org.apache.felix.scr.impl.manager.DependencyManager.setTargetFilter(DependencyManager.java:1939)
	at org.apache.felix.scr.impl.manager.AbstractComponentManager.updateTargets(AbstractComponentManager.java:1144)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.modify(SingleComponentManager.java:728)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.reconfigure(SingleComponentManager.java:645)
	at org.apache.felix.scr.impl.manager.SingleComponentManager.reconfigure(SingleComponentManager.java:609)
	at org.apache.felix.scr.impl.manager.ConfigurableComponentHolder.configurationUpdated(ConfigurableComponentHolder.java:426)
	at org.apache.felix.scr.impl.manager.RegionConfigurationSupport.configurationEvent(RegionConfigurationSupport.java:284)
	at org.apache.felix.scr.impl.manager.RegionConfigurationSupport$1.configurationEvent(RegionConfigurationSupport.java:89)
	at org.eclipse.equinox.internal.cm.EventDispatcher$1.run(EventDispatcher.java:89)
	at org.eclipse.equinox.internal.cm.SerializedTaskQueue$1.run(SerializedTaskQueue.java:36)
Caused by: org.osgi.framework.InvalidSyntaxException: Missing ')' at "pippo)": (&(objectClass=org.eclipse.kura.cloudconnection.publisher.CloudPublisher)pippo)
	at org.eclipse.osgi.internal.framework.FilterImpl$Parser.parse_filter(FilterImpl.java:1486)
	at org.eclipse.osgi.internal.framework.FilterImpl$Parser.parse(FilterImpl.java:1460)
	at org.eclipse.osgi.internal.framework.FilterImpl.newInstance(FilterImpl.java:147)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.createFilter(BundleContextImpl.java:977)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.getServiceReferences(ServiceRegistry.java:308)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.getServiceReferences(BundleContextImpl.java:535)
	at org.apache.felix.scr.impl.manager.ServiceTracker.getInitialReferences(ServiceTracker.java:241)
	at org.apache.felix.scr.impl.manager.ServiceTracker.open(ServiceTracker.java:216)
	... 12 more

```

--- a/kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/AbstractServicesUi.java
+++ b/kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/AbstractServicesUi.java
@@ -241,6 +241,7 @@ public abstract class AbstractServicesUi extends Composite {
         });
 
         if (param.getId().endsWith(TARGET_SUFFIX)) {
+            textBox.setReadOnly(true);
             String targetedService = param.getId().split(TARGET_SUFFIX)[0];
 
             DropDown dropDown = new DropDown();

